ambry-tools/src/main/java/com.github.ambry/store/BlobStatus.java

```diff
@@ -31,15 +30,21 @@
   private long earliestPutTimeMs = Utils.Infinite_Time;
   private long earliestDeleteTimeMs = Utils.Infinite_Time;
   private boolean isDeletedOrExpired;
-  private AtomicBoolean belongsToRecentIndexSegment = new AtomicBoolean(false);
+  private boolean belongsToRecentIndexSegment = false;
 
+  /**
+   * Initializes a {@link BlobStatus} with a list of Replica. ConsistencyChecker uses the {@link BlobStatus} to keep
+   * track of the status of a blob in every replica. "Replica" refers to a directory name where all replicas for a given
+   * partition is present.
+   * @param replicaList {@link List} of replicas for which blob status needs to be collected
+   */
   BlobStatus(List<String> replicaList) {
-    if (replicaList != null && replicaList.size() > 0) {
+    if (replicaList != null) {
       unavailable.addAll(replicaList);
     }
   }
 
-  Set<String> getAvailable() {
+  Set<String> getAvailableList() {
```

Comment:
Minor: These methods return sets instead of lists. Could they be named `getAvailableSet`?